src/components/input/input.js

```diff
@@ -76,11 +76,17 @@ function mdInputContainerDirective($mdTheming, $parse) {
 
   var INPUT_TAGS = ['INPUT', 'TEXTAREA', 'SELECT', 'MD-SELECT'];
 
+  var HIDDEN_INPUT_SELECTORS = ['[hidden=""]', '[hidden="true"]',
+                                '[aria-hidden=""]', '[aria-hidden="true"]',
+                                '[type="hidden"]',
+                                '.md-visually-hidden'];
```

Comment:
That indention style looks pretty weird here. Can you probably update and follow the patterns like in the `list` component for example?